app/mailers/assignment_mailer.rb

```diff
@@ -10,7 +10,7 @@ def notify(event, author, recipient, assigned)
       url = project_media.full_url
       title, media_title = project_media.title
       description = project_media.description
-      project = project_media.project
+      project = project_media.media_project
```

Comment:
If this is an assignment for project media, we don't need to mention project at all

Reply:
Done